feature-libs/asm/customer-360/components/customer-360-components.module.ts

```diff
@@ -0,0 +1,119 @@
+  AsmCustomerSavedCartComponent,
+} from './sections/components';
+import { AsmCustomerCouponComponent } from './sections/asm-customer-coupon/asm-customer-coupon.component';
+import { AsmCustomerCouponComponentModule } from './sections/asm-customer-coupon/asm-customer-coupon.module';
+import { AsmCustomerPromotionComponent } from './sections/asm-customer-promotion/asm-customer-promotion.component';
+import { AsmCustomerPromotionComponentModule } from './sections/asm-customer-promotion/asm-customer-promotion.module';
+import { AsmCustomerCustomerCouponComponent } from './sections/asm-customer-customer-coupon/asm-customer-customer-coupon.component';
+import { AsmCustomerCustomerCouponComponentModule } from './sections/asm-customer-customer-coupon/asm-customer-customer-coupon.module';
+
+@NgModule({
+  imports: [
+    CommonModule,
+    StarRatingModule,
+    I18nModule,
+    ArgsModule,
+    MediaModule,
+    IconModule,
+    KeyboardFocusModule,
+    PageComponentModule,
+    MessageComponentModule,
+    AsmCustomerActiveCartModule,
+    AsmCustomerProductInterestsModule,
+    AsmCustomerSavedCartModule,
+    AsmCustomerProfileModule,
+    AsmCustomerActivityModule,
+    AsmCustomerMapComponentModule,
+    AsmCustomerProductReviewsComponentModule,
+    AsmCustomerSupportTicketsComponentModule,
+    AsmCustomerCouponComponentModule,
+    AsmCustomerPromotionComponentModule,
+    AsmCustomerCustomerCouponComponentModule,
+    FeaturesConfigModule,
+  ],
+  declarations: [Customer360Component, AsmCustomerSectionComponent],
+  exports: [Customer360Component],
+  providers: [
+    provideDefaultConfig({
+      cmsComponents: {
+        AsmCustomer360Component: {
+          component: Customer360Component,
+        },
+        AsmCustomer360ProfileComponent: {
+          component: AsmCustomerProfileComponent,
+        },
+        AsmCustomer360ActiveCartComponent: {
+          component: AsmCustomerActiveCartComponent,
+        },
+        AsmCustomer360ProductInterestsComponent: {
+          component: AsmCustomerProductInterestsComponent,
+        },
+        AsmCustomer360ProductReviewsComponent: {
+          component: AsmCustomerProductReviewsComponent,
+        },
+        AsmCustomer360SupportTicketsComponent: {
+          component: AsmCustomerSupportTicketsComponent,
+        },
+        AsmCustomer360SavedCartComponent: {
+          component: AsmCustomerSavedCartComponent,
+        },
+        AsmCustomer360CustomerActivityComponent: {
+          component: AsmCustomerActivityComponent,
+        },
+        AsmCustomer360MapComponent: {
+          component: AsmCustomerMapComponent,
+        },
+        AsmCustomer360CouponComponent: {
+          component: AsmCustomerCouponComponent,
+        },
+        AsmCustomer360PromotionComponent: {
+          component: AsmCustomerPromotionComponent,
+        },
+        AsmCustomer360CustomerCouponComponent: {
+          component: AsmCustomerCustomerCouponComponent,
+        },
```

Comment:
Prerequisite question: [do we _really_ need to employ the CMS components mechanism for the purpose of Customer 360](https://github.com/SAP/spartacus/pull/17809/files#r1324461081)?

If we really need to use this mechanism, only then the following question applies:

[MEDIUM QUESTION]
I can see that CMS components types' names don't match the Angular implementation components names (and their selectors). Was it a deliberate choice to keep them different? Would you see a value in unifying the names of CMS component types, class names and their html selectors (`<cx-...>`)

It will be impossible to unify them after the release without a breaking change.

Reply:
All names unified in PR https://github.com/SAP/spartacus/pull/17878
Thanks Hakwoo!